src/components/screens/WelcomeScreen.tsx

```diff
@@ -72,9 +73,21 @@ export const WelcomeScreen: React.FC<WelcomeScreenProps> = ({ navigation }) => {
         <View>
           <StyledTermsText md secondary medium>
             {t("welcomeScreen.terms.byProceeding")}
-            <Text md medium url="https://stellar.org/terms-of-service">
+            <Text
+              md
+              medium
+              url={`${STELLAR_FOUNDATION_BASE_URL}/terms-of-service`}
```

Comment:
I think it'd be better to export the whole url as it'll be used in more than one place: WelcomeScreen and About screen on settings